bldsys/cmake/template/gradle/download_vvl.gradle

```diff
@@ -76,5 +76,6 @@ dependencies {
 }
 
 project.afterEvaluate{
+    project.getTasks().getByName("mergeDebugJniLibFolders").dependsOn(unzip)
```

Comment:
This is a note to myself to rethink how this is working.  Task dependencies in gradle don't need to be realized completely anymore in order to establish dependencies.  This closure forces gradle to completely define each task and then search for the same name.  This will slow the build down.

Reply:
I hear yea, I'm not a fan of gradle's complicated way of task dependency.  Definitely NOT a today change.